src/fiatExchanges/normalizeQuotes.ts

```diff
@@ -0,0 +1,194 @@
+import { FiatAccountType, FiatAccountTypeQuoteData } from '@fiatconnect/fiatconnect-types'
+import { FiatExchangeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { FiatConnectQuote } from 'src/fiatconnect'
+import {
+  SUPPORTED_FIAT_ACCOUNT_SCHEMAS,
+  SUPPORTED_FIAT_ACCOUNT_TYPES,
+} from 'src/fiatconnect/FiatDetailsScreen'
+import {
+  CICOFlow,
+  FetchProvidersOutput,
+  isSimplexQuote,
+  PaymentMethod,
+  SimplexQuote,
+} from 'src/fiatExchanges/utils'
+import i18n from 'src/i18n'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { navigateToURI } from 'src/utils/linking'
+
+export interface NormalizedQuote {
+  quote: QuoteInfo
+  provider: ProviderInfo
+}
+
+export interface QuoteInfo {
+  paymentMethod: PaymentMethod
+  fee: number | null
+  kycInfo: string | null
+  timeEstimation: string
+  onPress: () => void
+}
+
+export interface ProviderInfo {
+  name: string
+  logo: string
+  id: string
+}
+
+const strings = {
+  oneHour: i18n.t('selectProviderScreen.oneHour'),
+  numDays: i18n.t('selectProviderScreen.numDays'),
+  idRequired: i18n.t('selectProviderScreen.idRequired'),
+}
+
+function getSettlementEstimation(lower: string | undefined, upper: string | undefined) {
+  // TODO: Dynamically generate time estimation strings
+  //
+  return strings.numDays
+}
+
+export function normalizeQuotes(
+  flow: CICOFlow,
+  fiatConnectQuotes: FiatConnectQuote[] | undefined,
+  externalProviders: FetchProvidersOutput[] | undefined
+): NormalizedQuote[] {
+  return [
+    ...normalizeFiatConnectQuotes(flow, fiatConnectQuotes),
+    ...normalizeExternalProviders(flow, externalProviders),
+  ].sort(sortQuotesByFee)
+}
+
+export const sortQuotesByFee = (
+  { quote: quote1 }: NormalizedQuote,
+  { quote: quote2 }: NormalizedQuote
+) => {
+  const providerFee1 = quote1.fee ?? 0
+  const providerFee2 = quote2.fee ?? 0
+
+  return providerFee1 > providerFee2 ? 1 : -1
+}
+
+export function normalizeFiatConnectQuotes(
```

Comment:
I get that things like onPress can't be done in the cloud function, but I'm a little surprised that we're doing this much postprocessing after calling our cloud function, given that it's not in use yet. Seems like there might be an opportunity to just update the cloud function return types for greater ease of consumption-- in particular, for flattening out quotes with different fiat account schemas